docs/content/tools/_index.md

```diff
@@ -39,3 +39,4 @@ See the [NIST Software Disclaimer](https://www.nist.gov/disclaimer) for more inf
 | [Xacta 360](https://www.telos.com/offerings/xacta-360-continuous-compliance-assessment/) | Telos | Xacta 360 is a cyber risk management and compliance analytics platform that enables users to create and submit FedRAMP system security plans (SSPs) in OSCAL format. Future OSCAL capabilities are forthcoming as the standard evolves. | [license](https://cdn.telos.com/wp-content/uploads/2021/06/22150746/Xacta-360-EULA-US.pdf) |
 | [Atlasity: Continuous Compliance Automation](https://atlasity.io/partnership/) | C2 Labs | Atlasity CE (release 2.0) runs in any environment and supports the development of OSCAL v1.0 content for Catalogs, Profiles, System Security Plans and Components. Additional detail can be found in this blog post: [Atlasity Delivers Free Tools to Create OSCAL Content](https://www.c2labs.com/post/atlasity-delivers-free-tools-to-create-oscal-content). | community edition |
 | [control_freak](https://controlfreak.risk-redux.io/) | Risk Redux | This tool seeks to provide folks with a searchable and easy-to-navigate reference for NIST SP 800-53 Revision 5. It is [an open-source application from the Risk Redux project](https://github.com/risk-redux/control_freak), built using parsed content directly from the OSCAL repositories. | open-source |
+| [OSCAL4NEO4J](https://github.com/Agh42/oscal4neo4j) | The OSCAL4NEO4J Project | This project features a set of Neo4J cypher scripts. These will import OSCAL catalogs and profiles directly from the official Github-repositories into a Neo4J database. Once imported, the can be used in queries to gain insight into the structure of control catalogs and baselines. The project aims to add tool support for the implementation and assessment layers by allowing the documentation of the component model, the POA&M model as well as assessment-plans and -results. | open source |
```

Comment:
Please correct at minimum the typo in the  statement above: "the can" should read "they can". However, the statement is not very accurate. I assume the OSCAL catalogs and controls are used to generate OSCAL content (CDefs, SSP, AP, AS, POA&M) and not to change OSCAL models. Please consider updating the information for clarity. Here is some suggested text, but it requires your review for accuracy. 
"This project features a set of Neo4J cypher scripts which will import OSCAL catalogs and profiles directly from the official Github-repositories into a Neo4J database. Once imported, the information can be queried to gain insight into the structure of those control catalogs and baselines. The project aims to add tool support for the implementation and assessment layers by allowing to generate component definitions, system security plans, assessment-plans, assessment-results and POA&Ms." 
Please note, I added system security plan above assuming it has been omitted by mistake.

Reply:
Thank you for clarifying the terminology. You are correct of course - the goal is to create content based on the models. I updated the description as outlined by you.